roles/satellite-clone/tasks/main.yml

```diff
@@ -257,23 +257,9 @@
 
 - name: Finalize clone
   block:
-  # Cron services can interfere with restoring the backup
-  - name: stop crond
-    systemd:
-      name: crond
-      state: stopped
-
   - name: restore using foreman-maintain
     command: foreman-maintain restore --assumeyes {{ backup_dir }}
 
-  - name: start crond
-    systemd:
-      name: crond
-      state: started
-
-  - name: Restart services
-    command: foreman-maintain service start
-
     # We don't use pause module here because it displays a prompt
     # to use ctrl+c which would break out of a production script.
   - name: Wait for foreman-tasks service to start
```

Comment:
Is the sleep still needed? By now, don't we guarantee services being available after `systemctl start $service`?

Reply:
one hunk at a time ;)

foreman-tasks takes a moment to assemble, so `hammer ping` often shows that one as "dead" if you do it *directly* after, but the sleep is certainly too long